.profile.d/inject_react_app_env.sh

```diff
@@ -5,9 +5,17 @@ set -e
 # Debug, echo every command
 #set -x
 
-# Each bundle is generated with a unique hash name
-# to bust browser cache.
-js_bundle=/app/build/static/js/main.*.js
+static_json=/app/static.json
+if [ -f $static_json ]
+then
+  echo "Resolving static bundle from $static_json"
+  static_root=$(/app/.heroku/node/bin/node -pe 'JSON.parse(process.argv[1]).root || ""' "$(cat $static_json)")
```

Comment:
This parses to JSON with Node, which sets up a dependency on that buildpack. That is why the bundle injection script (below) is written in Ruby.

Please switch this to Ruby for JSON parsing.

Reply:
Why Ruby? Ruby is included and available by default in the Heroku stack image. Node is not, requires a buildpack.

Reply:
I have zero ruby experience, sorry.